v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -77,7 +62,37 @@ const responseBuilder = async (metadata, message, { Config }, payload) => {
 };
 
 const processTrackEvent = async (metadata, message, destination) => {
-  const payload = constructPayload(message, trackMapping);
+  let flag = 0;
+  const { Config } = destination;
+  const { event } = message;
+  const { listOfConversions } = Config;
+  for (let i = 0; i < listOfConversions.length; i += 1) {
+    if (listOfConversions[i].conversions === event) {
+      flag = 1;
+      break;
+    }
+  }
+  if (event === undefined || event === "" || flag === 0) {
+    throw new ErrorBuilder()
+      .setMessage(
+        `[Google_adwords_enhanced_conversions]:: Conversion named ${event} is not exist in rudderstack dashboard`
+      )
+      .setStatus(400)
+      .build();
+  }
+  const { requireHash } = destination.Config;
+  if (requireHash === false) {
+    message.integrations.google_adwords_enhanced_conversions = { hashed: true };
```

Comment:
why are we doing the opposite here? Seems like there's an issue with `handleMetadataForValue`

Reply:
I think it means, as the user is marking that no hashing is required, it means the desired fields are already hashed. And that is why hashed is marked as true. @ItsSudip correct me if I am wrong.

Reply:
Yes, @shrouti1507 you are correct. But changed the implementation so it is also changed.